lib/fog/libvirt/models/compute/templates/server.xml.erb

```diff
@@ -13,6 +13,23 @@
     <apic/>
     <pae/>
   </features>
+<% if !cpu.empty? -%>
+<% if cpu[:mode] -%>
```

Comment:
The situation is a bit messy (https://libvirt.org/formatdomain.html#elementsCPU) and for the sake for future-proveness (in theory more parameters could be supported) and setting default values I did it that way.